File: Spring Security Tutorials/05th_Endpoint_Authorization/src/main/java/com/example/fifth/controllers/DemoController.java
package com.example.fifth.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api")
public class DemoController {

    @GetMapping("/demo")
    public String demo() {
        return "Demo!";
    }

    @GetMapping("/hello")
    public String hello() {
        return "Hello!";
    }
}

File: Project/IDM/src/main/kotlin/com/project/idm/presentation/controllers/DemoController.kt
package com.project.idm.presentation.controllers

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class DemoController {

    @GetMapping("/hello")
    fun hello(): String {
        return "Hello!"
    }

    @GetMapping("/demo")
    fun demo(): String {
        return "Demo"
    }
}